Upstream: no
Reason: this test fails only on builders with "wc: standard input: Bad file descriptor"
diff --git a/slsh/lib/test/test_process.sl b/slsh/lib/test/test_process.sl
index d72b232..54d0796 100644
--- a/slsh/lib/test/test_process.sl
+++ b/slsh/lib/test/test_process.sl
@@ -37,6 +37,8 @@ private define test_process ()
 define slsh_main ()
 {
    start_test ("process");
+   () = fprintf(stdout, "Skipped\n");
+   return;
    test_process ();
    end_test ();
 }
